frontend/src/utils/server-utils.ts

```diff
@@ -207,7 +207,7 @@ export const getStaticRasterDataCoverage = (layer: StaticRasterLayerProps) => {
     return [];
   }
   // raw data comes in as {"dates": ["YYYY-MM-DD"]}
-  return dates.map(v => moment(v, 'YYYY-MM-DD').valueOf());
+  return dates.map(v => moment.utc(v, 'YYYY-MM-DD').valueOf());
```

Comment:
why make the change here but not on line `201`?

Reply:
Is it worth doing what we do for other layers?
`.map(item => moment.utc(item.date).set({ hour: 12, minute: 0 }).valueOf())`